bin/test-task.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+"use strict";
+
+var util = require("util");
+
+var assert = require("assert"),
+    async = require("async"),
+    fs = require("fs"),
+    tmp = require("tmp"),
+    env = require("require-env");
+
+var worker = require("../lib/worker"),
+    upload = require("../lib/localUploader.js");
```

Comment:
File systems _might_ be case-sensitive (or may just claim to be), so I prefer `_`s in filenames, even if their exports make sense as camel case.